server/controllers/entities/lib/move_to_wikidata.ts

```diff
@@ -24,9 +27,25 @@ export async function moveInvEntityToWikidata (user: User, invEntityUri: InvEnti
 
   let claims, labels
   if ('labels' in entity) labels = entity.labels
-  if ('claims' in entity) claims = extendInvClaims(entity.claims)
+  if ('claims' in entity) claims = expandInvClaims(entity.claims)
 
-  const { uri: wdEntityUri } = await createWdEntity({ labels, claims, user, isAlreadyValidated: true })
+  const conflictingWdEntities = await resolveExternalIds(claims, {
+    resolveOnWikidata: true,
+    resolveLocally: false,
+    refresh: true,
+  })
+  if (conflictingWdEntities?.length > 0) {
+    throw newError('Can not move to Wikidata: some Wikidata entities share the same identifiers', 400, { conflicts: conflictingWdEntities })
+  }
+
+  // Local claims will be preserved in a local layer during merge
+  const claimsWithoutLocalClaims = omitBy(claims, (propertyClaims, property) => isInvPropertyUri(property))
+  const { uri: wdEntityUri } = await createWdEntity({
```

Comment:
What about a `try...catch` around this function which touches an external service ? 
Otherwise, `mergeEntities` below will crash with some `undefined` `wdEntityUri` which may be poor for the nice admin who will review logs later on

Reply:
If `createWdEntity` throws an error, `mergeEntities` will not be called, which is what we want, no? Did I get it wrong?